CadreDecisionnelAlerteCOVID.md

```diff
@@ -0,0 +1,210 @@
+
+## Qu’est-ce qu’un backlog de produit agile? 
+
+Un backlog de produits est un outil utilisé dans le contexte du développement agile pour suivre l’avancée d’un produit, d’en faire le suivi et d’évaluer, entre autres, les nouvelles fonctionnalités, les changements à des fonctionnalités existantes, les correctifs de bogues, les changements à l’infrastructure, ou suivre le développement d’autres activités qu’une équipe pourrait accomplir afin d’atteindre un résultat escompté.
+
+*   **Flexible et constamment en changement :** Au fur et à mesure que nous apprenons de nouvelles choses, la priorité et l’ordre des éléments en attente peuvent changer.
+*   **Fondé sur les besoins :** Les éléments dans ce backlog sont basés sur les besoins de santé publique et des personnes utilisant l’application. 
+*   **Défini par niveau de priorité, et non par des échéances fixes :** Nous publions les fonctionnalités lorsqu’elles sont prêtes – ni plus tôt ni plus tard. 
+
+Ce backlog est réévalué régulièrement, et sera mis à jour toutes les deux semaines. Cette fréquence est harmonisée avec les rencontres « sprints » de l’équipe. 
+
+## Gouvernance et processus
+
+La gouvernance du service de notification des expositions est partagée entre plusieurs organes directeurs, et est gérée conjointement entre Santé Canada et le Service numérique canadien :
+
+<table>
+  <tr>
+   <td><strong>Organe directeur</strong>
+   </td>
+   <td><strong>Rôle</strong>
+   </td>
+   <td><strong>Fréquence</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>Conseillers médicaux exécutifs de l’ASPC
+   </td>
+   <td><p><strong>Quoi :</strong> Conseiller médical principal pour Alerte COVID et intermédiaire pour le Comité consultatif spécial.
+   </p>
+   </td>
+   <td>Au besoin
+   </td>
+  </tr>
+  <tr>
+   <td>Comité consultatif spécial (CCS)
+   </td>
+   <td><p><strong>Qui :</strong> Les coprésidents du Réseau pancanadien de santé publique (RPS) comprennent des représentants de plusieurs ministères du gouvernement du Canada, ainsi que des membres du Conseil du Réseau de santé publique et du Conseil des médecins hygiénistes en chef. La Dre Theresa Tam, administratrice en chef de la santé publique du Canada, et le Dr Saqib Shahab, médecin hygiéniste en chef de la Saskatchewan, coprésident le comité.
+   </p>
+<p>
+<strong>Quoi :</strong> Fournir conseil à la Conférence des sous-ministres de la santé FPT sur la coordination, la politique de santé publique et le contenu technique liés à la pandémie de COVID-19.
+</p>
+   </td>
+   <td>Hebdomadaire
+   </td>
+  </tr>
+  <tr>
+   <td>Groupe de travail sur la santé publique fédéral / provincial / territorial (FPT)
+   </td>
+   <td><strong><p>Qui :</strong> Groupe de travail composé de représentants de toutes les équipes provinciales et territoriales d’Alerte COVID (y compris des représentants de la santé publique et de la prestation de services numériques)
```

Comment:
```suggestion
   <td><p><strong>Qui :</strong> Groupe de travail composé de représentants de toutes les équipes provinciales et territoriales d’Alerte COVID (y compris des représentants de la santé publique et de la prestation de services numériques)
```

Reply:
Really appreciate it – thanks @davidbgk ! 🎉